### PR TITLE
Join table cat n service req

### PR DESCRIPTION
**Pull request by 1CIM**
- Create Bids model

### CODE REVIEW

File: db/migrate/20211029072510_create_bids.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class CreateBids < ActiveRecord::Migration[6.1]
+  def change
+    create_table :bids do |t|
+      t.float :amount
+      t.references :service_request, foreign_key: true
+      t.references :creator, index: true,  foreign_key: {to_table: :users}
```
Comment (offcast-dev) [inline db/migrate/20211029072510_create_bids.rb:6]: do we have a creator?

**Review by offcast-dev — COMMENTED** [review]
question

File: db/migrate/20211029072510_create_bids.rb (hunk 1 above)
Comment (1CIM) [reply db/migrate/20211029072510_create_bids.rb:6]: Now it does

**Review by offcast-dev — APPROVED** [review]
I guess it is ok, not sure about creator stuff, but is just nit familiar syntax

**Pull request merged**